src/components/PeopleTable.tsx

```diff
@@ -576,70 +1,283 @@
+/* eslint-disable @typescript-eslint/no-unused-expressions */
+import { useEffect, useState } from 'react';
+import { getPeople } from '../api';
+import { Loader } from './Loader';
+import { Link, useLocation, useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+
+type People = {
+  name: string;
+  sex: string;
+  born: number;
+  died: number;
+  motherName: string | null;
+  fatherName: string | null;
+  slug: string;
+};
 
-        <tr data-cy="person">
-          <td>
-            <a href="#/people/carolus-haverbeke-1832">Carolus Haverbeke</a>
-          </td>
-          <td>m</td>
-          <td>1832</td>
-          <td>1905</td>
-          <td>
-            <a
-              className="has-text-danger"
-              href="#/people/maria-van-brussel-1801"
-            >
-              Maria van Brussel
-            </a>
-          </td>
-          <td>
-            <a href="#/people/carel-haverbeke-1796">Carel Haverbeke</a>
-          </td>
-        </tr>
+type SortByWhat = 'sex' | 'born' | 'died' | 'name';
 
-        <tr data-cy="person">
-          <td>
-            <a className="has-text-danger" href="#/people/maria-sturm-1835">
-              Maria Sturm
-            </a>
-          </td>
-          <td>f</td>
-          <td>1835</td>
-          <td>1917</td>
-          <td>Seraphina Spelier</td>
-          <td>Charles Sturm</td>
-        </tr>
+export const PeopleTable = () => {
+  const [searchParams] = useSearchParams();
+  const [people, setPeople] = useState<People[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const { pathname } = useLocation();
+  const [filteredPeople, setFilteredPeople] = useState<People[]>([]);
+  const [currentIsReverse, setCurrentIsReverse] = useState<boolean>();
+  const [currentSortParam, setCurrentSortParam] = useState('');
+  const [errorText, setErrorTest] = useState('');
```

Comment:
There is a typo in the state variable name `setErrorTest`. It should be `setErrorText` to match the intended purpose of storing error messages.